package-lock.json

```diff
@@ -17092,9 +17092,9 @@
       "dev": true
     },
     "node_modules/semver": {
-      "version": "5.7.0",
-      "resolved": "https://registry.npmjs.org/semver/-/semver-5.7.0.tgz",
-      "integrity": "sha512-Ya52jSX2u7QKghxeoFGpLwCtGlt7j0oY9DYb5apt9nPlJ42ID+ulTXESnt/qAQcoSERyZ5sl3LDIOw0nAn/5DA==",
+      "version": "5.7.2",
```

Comment:
Is it our intention to keep imperative's direct dependency at the back-level version of 5.7.2?

Is that old enough that it doe not have the vulnerability?

Reply:
Yes, it was patched in 5.7.2. I did not want to increase the major version to 7.5.4 just in case it would break something.